lib/sys.js

```diff
@@ -24,6 +24,8 @@
 // The sys module was renamed to 'util'. This shim remains to keep old programs
 // working. `sys` is deprecated and shouldn't be used.
 
+// Note to maintainers: Although this module has been deprecated for a while
+// we do not plan to remove it. See: https://github.com/nodejs/node/pull/35407#issuecomment-700693439
```

Comment:
```suggestion
// Note to maintainers: Although this module has been deprecated for a while,
// there are currently no plans to remove it.
// See: https://github.com/nodejs/node/pull/35407#issuecomment-700693439
```

Reply:
I'm not sure your suggestion is correct, there are also plans to remove it, but they never get consensus, cf the trail of closed PRs that attempted to do that. What we need to stress out is that there are plans to never remove it.

Reply:
What wording would you suggest then?